explainer.md

```diff
@@ -346,13 +522,88 @@ of custom-drawn APIs.
 On the web, this would allow creating an accessible solution to canvas-based UI
 which does not rely on fallback or visually-hidden DOM content.
 
-The API for Virtual Accessibility Nodes will be specified at a later date.
+Constructing a node can be achieved simply by creating an AccessibleNode
```

Comment:
I guess my hope was not to bikeshed details but to capture an approximation
of the direction we were going, knowing that those details are far more in
flux.

We shouldn't wait to figure out the outline for other phases now because it
impacts the first phase. For example virtual nodes implies that IDREFs need
to be replaced with references to AccessibleNodes, not DOM Elements